### PR TITLE
Add brigmedic belt

### PR DESCRIPTION
**Pull request by fqqf**
## Short description
image

### Brigmedic belt
https://github.com/user-attachments/assets/cadb45b3-0f86-4c1f-8bc6-ba7ac46d6a90

- Brigmedic belt will fit both med and sec stuff. Exclusive to brigmedic role only, others belt removed from loadout

## Checks

- [x] I do not require assistance to complete the PR.
- [x] Before posting/requesting review of a PR, I have verified that the changes work.
- [x] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [x] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**

:cl: Miracula
- add: Added brigmedic belt

### CODE REVIEW

**Comment by fqqf** [comment]
ready for review

File: Resources/Prototypes/_StarLight/Entities/Clothing/Belt/belts.yml [hunk 1]
```diff
@@ -148,4 +148,46 @@
...
+        - CigPack
+        - Taser
+        - Wrench
+        - Bottle
+        - Spray
+        - Brutepack
+        - Bloodpack
+        - Gauze
+        - Ointment
+        - CigPack
```
Comment (StarlightHost) [inline Resources/Prototypes/_StarLight/Entities/Clothing/Belt/belts.yml]: ```- CigPack``` duplicate

**Review by StarlightHost — APPROVED** [review]

File: Resources/Prototypes/_StarLight/Entities/Clothing/Belt/belts.yml (hunk 1 above)
Comment (fqqf) [reply Resources/Prototypes/_StarLight/Entities/Clothing/Belt/belts.yml]: fixed

**Comment by StarlightHost** [comment]
image

**Pull request merged**